src/components/organisms/radio-group/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group"
+import clsx from "clsx"
+import React from "react"
+
+type RadioGroupProps = RadioGroupPrimitive.RadioGroupProps &
+  React.RefAttributes<HTMLDivElement>
+
+type RadioGroupItemProps = {
+  label: string
+  sublabel?: string
+  description?: string
+} & RadioGroupPrimitive.RadioGroupItemProps &
+  React.RefAttributes<HTMLButtonElement>
+
+const RadioGroup: React.FC<RadioGroupProps> & {
+  Item: React.FC<RadioGroupItemProps>
+} = ({ children, ...rest }) => {
+  return (
+    <RadioGroupPrimitive.RadioGroup {...rest}>
+      {children}
+    </RadioGroupPrimitive.RadioGroup>
+  )
+}
+
+const Item = ({
+  label,
+  sublabel,
+  description,
+  ...rest
+}: RadioGroupItemProps) => {
+  return (
+    <div
+      className={clsx(
+        "rounded-base border border-grey-20 p-base flex items-baseline mb-xsmall last:mb-0 gap-base"
+      )}
+    >
+      <RadioGroupPrimitive.Item
+        {...rest}
+        id={rest.value}
+        className={clsx(
+          "radio-outer-ring outline-0",
+          "shrink-0 w-[20px] h-[20px] shadow-[0_0_0_1px] shadow-[#D1D5DB] rounded-circle",
+        )}
+      >
+        <RadioGroupPrimitive.Indicator
```

Comment:
this seems to be misaligned (I have made the same comment for Kasper in his pr), just fyi that it should be fixed, i see you are waiting for his pr to be merged first

Reply:
right! made a tiny improvement by changing the `align-items` property from `baseline` to `flex-start` and used negative margins to kinda "pull up" the div containing the label + description. It might not be pixel perfect aligned but let's investigate that later down the road if that's okay with you :+1: What do you think?

Reply:
Just a quick comment on this @zakariaelas, Ive fixed it in my PR by simply changing `items-baseline` to `items-start` on the RadioGroup.Item component. That way we dont need to do any magic negative margins

Reply:
alrighty @kasperkristensen, that sounds perfect!